docker/packager/scripts/example.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
So yes, we have a directory `scripts`, let's rename it to `pre-build` and check its existence.